model/command_config.go

```diff
@@ -134,7 +135,8 @@ func (c *CommandConfig) UpdateImportPath() error {
 				if strings.HasPrefix(currentPath, path) && len(currentPath) > len(path)+1 {
 					importPath = currentPath[len(path)+1:]
 					// Remove the source from the path if it is there
-					if len(importPath) > 4 && strings.ToLower(importPath[0:4]) == "src/" {
+					isSRC := strings.ToLower(importPath[0:4])
```

Comment:
We are missing the check to see that len(importPath)>4 can you do that before setting the isSRC variable ?
Otherwise looks good